src/napari_micromanager/_core_link.py

```diff
@@ -32,42 +35,46 @@ def __init__(
         self._mda_handler = _NapariMDAHandler(self._mmc, viewer)
         self._live_timer_id: int | None = None
 
+        self.streaming_timer = QTimer(parent=self)
```

Comment:
I'm curious if you determined that using an independent `QTimer` instead of `self.startTimer` was sufficient to resolve the slow down?  I ask because this is a bit surprising to me.  Before accepting this change, I'd prefer to know *why* it matters.  Looking into the Qt docs for hints if need be.  If true, it might have to do with threading or something... but lets figure out why we're doing it